networking/cidr-range-definitions.adoc

```diff
@@ -1,7 +1,12 @@
 :_content-type: ASSEMBLY
 [id="cidr-range-definitions"]
 = CIDR range definitions
+ifndef::openshift-rosa,openshift-dedicated[]
```

Comment:
I didn't see that this file was used in anything other than ROSA and OSD topic maps - just checking if this ifdef is really necessary (should the dedicated one just replace the common-attributes one)?